modules/server/src/topics/actionkeybindtopic.cpp

```diff
@@ -117,28 +133,21 @@ nlohmann::json ShortcutTopic::shortcutJson(const std::string& identifier) const
     }
     interaction::Action action = *found;
 
-    json.push_back({
-        { "identifier", action.identifier },
-        { "name", action.name },
-        { "synchronization", static_cast<bool>(!action.isLocal) },
-        { "documentation", action.documentation },
-        { "guiPath", action.guiPath },
-    });
-    return json;
+    return jsonAction(action);
 }
 
-void ShortcutTopic::sendData(nlohmann::json data) const {
-    _connection->sendJson(wrappedPayload({ { "shortcuts", data } }));
+void ActionKeybindTopic::sendData(nlohmann::json data) const {
+    _connection->sendJson(wrappedPayload({ data }));
```

Comment:
```suggestion
    nlohmann::json payload = wrappedPayload({ data });
    _connection->sendJson(std::move(payload));
```

@engbergandreas  Might agree with how annoying it can be to try to debug these issues ;)